### PR TITLE
VS-1514 backwards compatibility

### PR DESCRIPTION
**Pull request by gbggrant**
This PR makes the GvsPrepareRangesCallset wdl backwards compatible with older (well, at least Delta) call sets.
- It checks for the `call_PS` column in the vet table and if present uses phasing columns. If not present (as in Delta) does NOT include phasing.

Successful run again Delta data [here](https://app.terra.bio/#workspaces/allofus-drc-wgs-dev/GVS%20AoU%20Echo%20RD/job_history/4e65386a-e14e-42f7-928b-764c6b937512)
Successful integration test run [here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Integration/job_history/b4d62c02-59fa-4546-b34e-313e984635a7)

### CODE REVIEW

File: scripts/variantstore/scripts/create_ranges_cohort_extract_data_table.py [hunk 1]
```diff
@@ -425,6 +439,9 @@ def make_extract_table(call_set_identifier,
 
     args = parser.parse_args()
 
+    if not args.vet_ranges_extract_table_version:
```
Comment (mcovarr) [inline scripts/variantstore/scripts/create_ranges_cohort_extract_data_table.py:442]: Either here or in the parser we might want to validate that we actually got `V1`, `V2`, or nothing at all

Comment (gbggrant) [reply scripts/variantstore/scripts/create_ranges_cohort_extract_data_table.py:442]: Yeah, that's a good point. Will do!

**Review by RoriCremer — APPROVED** [review]

**Comment by gbggrant** [comment]
@mcovarr - I have added the validation for `V1` or `V2` into the python script

**Review by koncheto-broad — APPROVED** [review]
LGTM

**Pull request merged**